redwood-treehouse-host/src/iosMain/kotlin/app/cash/redwood/treehouse/TreehouseUIKitView.kt

```diff
@@ -72,16 +75,21 @@ public class TreehouseUIKitView(
   }
 
   internal fun updateUiConfiguration() {
-    mutableUiConfiguration.value = computeUiConfiguration(view.traitCollection)
+    mutableUiConfiguration.value = computeUiConfiguration(
+      traitCollection = view.traitCollection,
+      bounds = view.bounds,
+    )
   }
 }
 
 private fun computeUiConfiguration(
   traitCollection: UITraitCollection,
+  bounds: CValue<CGRect>,
 ): UiConfiguration {
   return UiConfiguration(
     darkMode = traitCollection.userInterfaceStyle == UIUserInterfaceStyleDark,
     safeAreaInsets = computeSafeAreaInsets(),
+    viewportSize = bounds.useContents { Size(size.width.toInt(), size.height.toInt()) },
```

Comment:
I think we should probably have `Size` use `Double`s (or `Dp`) instead of `Int`s. Unfortunately, iOS's coordinate system using `Double`s means we either use decimals everywhere or lose precision.